app/client-page/head.js

```diff
@@ -1,4 +1,4 @@
-import { useTranslation } from '../i18n'
+import { useTranslation } from 'react-i18next'
```

Comment:
I'm using this hook directly from the library, since the `i18next` instance is always initialised in a root `layout`, so you don't have to re-export it like in original examle.